internal/service/cloudprovideraccess/resource_cloud_provider_access_authorization_test.go

```diff
@@ -55,17 +57,31 @@ func basicAuthorizationTestCase(tb testing.TB) *resource.TestCase {
 		CheckDestroy:             checkDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: configAuthorizationAWS(projectID, policyName, roleName),
+				Config: configAuthorizationAWS(projectID, policyName, roleName, federatedDatabaseInstanceName, testS3Bucket),
```

Comment:
can we add some attribute checks for this resource in the test?

Reply:
thanks for catching that!